scripts/postinstall.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs')
+const path = require('path')
+
+if (process.platform !== 'win32') {
+  console.log('postinstall: not windows, so skipping!')
+  process.exit(0)
+}
+
+const from = path.resolve(
+  __dirname,
+  '..',
+  'deltachat-core-rust',
+  'target',
+  'release',
+  'deltachat.dll'
+)
+
+const getDestination = () => {
+  const argv = process.argv
+  if (argv.length === 3 && argv[2] === '--prebuild') {
+    return path.resolve(
+      __dirname,
+      '..',
+      'prebuilds',
+      'win32-x64',
+      'deltachat.dll'
+    )
+  } else {
+    return path.resolve(
+      __dirname,
+      '..',
+      'build',
+      'Release',
+      'deltachat.dll'
+    )
+  }
+}
+
+const dest = getDestination()
+
+copy(from, dest, (err) => {
```

Comment:
Why is copying only needed on windows?

Reply:
Because we need `deltachat.dll` from `deltachat-core-rust`.

Reply:
But isn't this the same on linux/mac, only difference is that we need a .so/.dylib?

Reply:
No since we're linking statically on linux and mac.

Reply:
Oh okay thanks :)